src/style.css

```
@@ -1,3 +1,112 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
+  --color-blue: rgba(0, 172, 220, 1);
+  --color-black: rgba(6, 11, 53, 1);
+  --color-grey: rgba(97, 96, 112, 1);
+
+}
+
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  margin: auto;
+  padding: 33px 17px 17px;
+  box-sizing: border-box;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 35px auto 0;
```

please, check correct indent from figma
```suggestion
  margin-top: 40px;
```